pkg/query/assetsapi.go

```diff
@@ -0,0 +1,104 @@
+package query
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/kubecost/cost-model/pkg/kubecost"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type assetResponse struct {
+	Code int                 `json:"code"`
+	Data []map[string]Assets `json:"data"`
+}
+
+type AssetParameters struct {
+	RestConfig *rest.Config
+	Ctx        context.Context
+
+	KubecostNamespace  string
+	ServiceName        string
+	Window             string
+	Aggregate          string
+	DisableAdjustments bool
+	Accumulate         bool
+	UseProxy           bool
+	FilterTypes        string
+}
+
+// QueryAssets queries /model/assets by proxying a request to Kubecost
+// through the Kubernetes API server if useProxy is true or, if it isn't, by
+// temporarily port forwarding to a Kubecost pod.
+func QueryAssets(p AssetParameters) ([]map[string]Assets, error) {
+
+	// aggregate, accumulate, and disableAdjustments are hardcoded;
+	// as other asset types are added in to be filtered by, this may change,
+	// but for now anything beyond isn't needed.
+
+	requestParams := map[string]string{
+		"window":             p.Window,
+		"aggregate":          "",
+		"accumulate":         "true",
+		"disableAdjustments": "true",
```

Comment:
I don't think this should default to `true`.

Reply:
Without `accumulate` being set to true, the result returned from the API is a time series, which doesn't seem useful if the end goal is single cost numbers.

Reply:
Furthermore (and this is really the relevant part...) the code also doesn't work when it's a time series.

Reply:
Accumulate should definitely be true, I was referring to `disableAdjustments`.